client/blocks/signup-form/style.scss

```diff
@@ -295,41 +267,6 @@ body.is-section-accept-invite {
 				display: flex;
 				flex-direction: column;
 				width: 100%;
-
-				.social-buttons__button {
```

Comment:
These styles are now centralized in `client/components/social-buttons/style.scss`